regression/cbmc-viewer/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+if(NOT WIN32)
```

Comment:
This seems to be redundant, as this is already checked in the parent CMakeLists. Remove one of the conditionals please.